.env-dist

```diff
@@ -29,7 +29,7 @@ SOCKETLABS_SECRET_KEY="dummy-value"
 SOCKETLABS_API_KEY="dummy-value"
 SOCKETLABS_VALIDATION_KEY="dummy-value"
 RELAY_FROM_ADDRESS="relay@127.0.0.1:8000"
-NEW_RELAY_FROM_ADDRESS="new_from@127.0.0.1:8000"
+NEW_RELAY_FROM_ADDRESS="new_from@127.0.0.1:8000"  # TODO: remove
```

Comment:
Removing this line breaks the migrations tests. We can remove it when this code goes to production.